c/ckb_consts.h

```diff
@@ -8,6 +8,7 @@
 #define SYS_ckb_load_header 2072
 #define SYS_ckb_load_input 2073
 #define SYS_ckb_load_witness 2074
+#define SYS_ckb_load_script 2075
```

Comment:
Can we change the number of this syscall to 2063? The syscalls are organized this way:

* 206x: general and current script group related syscalls
* 207x: transaction component load
* 208x: transaction component load by field
* 209x: cell data load

Reply:
Updated, please review again, thanks.